frontend/src/container/NewDashboard/DashboardDescription/Description.styles.scss

```diff
@@ -58,6 +58,7 @@
 		display: flex;
 		justify-content: space-between;
 		align-items: center;
+		padding-right: 16px;
```

Comment:
should we add padding or margin?

Reply:
I used padding as I didn't see the use of pushing the borders of the header inwards using margin, with padding the header's border will remain there and we will add space inside it.
also, the content below follows the same.

<img width="1723" alt="image" src="https://github.com/SigNoz/signoz/assets/162284829/94249a2e-3643-4a18-99be-38283596ad71">

<img width="1677" alt="image" src="https://github.com/SigNoz/signoz/assets/162284829/dda13c3c-3408-479d-9cd6-ae4b024b8f53">

Reply:
didn't get you how is this causing any issues ? 
![image](https://github.com/SigNoz/signoz/assets/54737045/7abb237c-4afe-4f9e-bcc1-9b8124612a78)

Reply:
Neither of them won't cause any issues, but I got your point about using margin to properly position the containers inside the containers.

Reply:
https://mailchimp.com/resources/padding-vs-margin/ for more details